tests/test_sessions.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
Please use single quotes wherever possible to match the rest of the repository

Reply:
replaced " with '

Reply:
Replaced